server/src/main/java/org/apache/iotdb/db/writelog/recover/LogReplayer.java

```diff
@@ -91,17 +91,24 @@ public void replayLogs() {
         logNodePrefix + FSFactoryProducer.getFSFactory().getFile(insertFilePath).getName());
 
     ILogReader logReader = logNode.getLogReader();
+    long readLogTime = 0;
+    long replayLogTime = 0;
     try {
       while (logReader.hasNext()) {
         try {
+          long start = System.nanoTime();
```

Comment:
System.nanoTime() will call "clock_gettime" which is a system call. So calculating the replay time of every row may have some performance impact.